### DISCUSSION TITLE
Default password when building myself

### DISCUSSION THREAD

**Discussion by nilo85** [Joshua-Riek/ubuntu-rockchip#736] [Q&A] [answered]
Hello, I just build the server image myself
`sudo ./build.sh --board=orangepi-5-plus --release=jammy  -so`
However it refuses to accept ubuntu/ubuntu at login, did I do something wrong?

**Comment by Joshua-Riek** [comment]
Cloud init shenanigans most likely, since there is no pre-made user on the system could-init will create the default user during boot. 

You may need to wait a few minutes as cloud-init can be slow. Or there is a bug with Jammy images, I've been to focused on Noble and have not tested Jammy in some time.

**Reply by nilo85** [reply]
Interesting, never had the issue with the prebuilt images, maybe I pulled a broken commit the other day =) I got some error about some armor service so maybe I should stash my tweaks to kernel config and retry with a fresh pull of the builder scripts

**Reply by Joshua-Riek** [reply]
After I finish the release for Noble, i'm going to work on Jammy again to ensure everything is working properly.

**Comment by ohyeah521** [comment, marked as answer]
After many tests, this is a commit that works fine. 1c6aef87489040d58dbd688e10b38d38a556680e

**Comment by nilo85** [comment]
Great thanks! you both =)

After stable noble I suspect jammy is not as interesting anymore but probably does not hurt to have it supported as long as its within its lifecycle =)
